client/src/landingPageContent/JoinUs.js

```diff
@@ -0,0 +1,129 @@
+import React, { useState, useEffect } from 'react';
+import { Container, Statistic, Transition, Segment, Step, Icon, Header, Image } from 'semantic-ui-react'
+import { makeCall } from '../apis';
+import {black, darkBlack, grey} from "../colors"
+
+export default function JoinUs (props) {
+    const [counts, setCounts] = useState({})
+    const [sendingRequest, setSendingRequest] = useState(false)
+    const fetchCounts = () => {
```

Comment:
Can you add a line of documentation about what this function does? Not immediately clear what it's needed for at first glance if I'm not familiar with the backend call

Reply:
addressed!